completions/minicom

```diff
@@ -34,7 +34,7 @@ _minicom()
     fi
 
     local -a files
-    _comp_expand_glob files '/etc/minirc.* /etc/minicom/minirc.* ~/.minirc.*'
+    _comp_expand_glob files '{/etc/,/etc/minicom/,~/.}minirc.?*'
```

Comment:
If I read correctly, the empty completions would occur (only) with files having basename `minirc.`. If that's the case, could note that in the commit message.

Reply:
Sure!

Reply:
I added it in 3597289e71be0ddedc9a1eea590e72a67db9204f.